### PR TITLE
Update manifest.json

### PR DESCRIPTION
**Pull request by maxcanna**

### CODE REVIEW

**Comment by maxcanna** [comment]
[Add manifest.json](https://trello.com/c/samW8GCE/77-add-manifestjson)

File: public/manifest.json [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+          "src": "/favicon-512x512.png",
+          "sizes": "512x512",
+          "type": "image/png"
+      }
+  ],
   "display": "standalone",
   "gcm_sender_id": "482941778795",
   "orientation": "any",
-  "theme_color": "#54b4eb",
+  "theme_color": "#e95420",
```
Comment (demerzel3) [inline public/manifest.json]: Our theme color is `#f96232`

File: public/index.html [hunk 2]
```diff
@@ -3,10 +3,12 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#000000">
+    <meta name="theme-color" content="#e95420">
```
Comment (demerzel3) [inline public/index.html]: Our theme color is `#f96232`. Also is there a `background-color` meta as well? If so please add it with `white` as in the manifest.

Comment (maxcanna) [reply public/index.html]: Thank you! I didn't know the exact value so I grabbed it from the website. Gonna update that too.

No, that's the only meta. There are a bunch of other one iOS specific but we can add them on the future

File: public/manifest.json (hunk 1 above)
Comment (maxcanna) [reply public/manifest.json]: Will change. Thanks for the tip

**Pull request merged**